ci/pipeline.yml

```diff
@@ -85,7 +85,7 @@ jobs:
   plan:
   - in_parallel:
     - get: broker-src
-      passed: [acceptance-tests-staging]
+      passed: [push-cf-cdn-service-broker-staging]
```

Comment:
Is this is renaming to the new service?

Reply:
Answered in https://github.com/cloud-gov/cf-domain-broker-alb/pull/35 - thank you!